app/src/main/java/xterminators/spellingbee/cli/CLIView.java

```diff
@@ -0,0 +1,180 @@
+package xterminators.spellingbee.cli;
+
+import java.util.Arrays;
+import java.util.List;
+
+import xterminators.spellingbee.model.Rank;
+
+/**
+ * The view of the CLI mode of the Spelling Bee game. This class takes
+ * instructions and data from the controller and displays output accordingly.
+ */
+public class CLIView {
+    public static final String ANSI_BOLD = "\u001B[1m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_RESET = "\u001B[0m";
+
+    /**
+     * Displays the given list of found words for the puzzle.
+     * 
+     * @param foundWords The list of found words
+     */
+    public void showFoundWords(List<String> foundWords) {
+        if (foundWords.isEmpty()) {
+            System.out.println("You have not found any words yet.");
+        } else if (foundWords.size() == 1) {
+            System.out.println("You have found 1 word:");
+        } else {
+            System.out.println("You have found " + foundWords.size() + " words:");
+        }
```

Comment:
Why is there a different print statement when they have found 1 word and > 1 word?

Reply:
Singular vs plural. So "found 1 word" vs "found n word**s**".